scalability_and_performance/index.adoc

```diff
@@ -0,0 +1,63 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="index"]
+= {product-title} Scalability and Performance overview
+include::_attributes/common-attributes.adoc[]
+:context: index
+
+// ifndef::openshift-dedicated,openshift-rosa[]
+{product-title} offers best practices and utilities to help you optimize the performance and scale of your {product-title} clusters. This book in the documentation provides information on recommended performance and scalability practices, reference design specifications, optimization and low latency tuning.
+
+To get support on these topics, refer to the section of the documentation entitled xref:../support/getting-support.adoc#getting-support[Getting support]
```

Comment:
I'd restructure this sentence a bit: 

"To engage with Red Hat support, see [Getting support]." (Don't forget the period at the end, too!)

Reply:
done, thank you.